### ISSUE TITLE
SPC doesn't open the menu

### ISSUE THREAD

**Issue by simendsjo** [syl20bnr/spacemacs#5531]
#### Description
Pressing SPC doesn't open the menu. Neither does other compinations (`SPC f` etc). Other commands, such as `M-x`, opens menus.

#### Reproduction guide
- Start Emacs
- Press SPC

*Observed behaviour:*
No menu opens

*Expected behaviour:*
Menu should open

#### System Info
- OS: windows-nt
- Emacs: 25.0.50.1
- Spacemacs: 0.105.14
- Spacemacs branch: master (rev. a298a5a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion better-defaults emacs-lisp git fsharp markdown org mu4e spell-checking syntax-checking version-control)
```

**Comment by simendsjo** [comment]
Is there any logging I can add to give an error message? The menu never opens, so it's very easy to reproduce for me :)

**Comment by StreakyCobra** [comment]
Does `M-m` open it?
Does keybinding work even if the menu doesn't show up? (you can try to check for `SPC b h` going to the `*spacemacs*` buffer and `SPC b s` going to the `*scratch*` buffer)

**Comment by simendsjo** [comment]
I can check `M-m` tomorrow, but I suspect it's working. All keybindings seems to work, and `SPC b b` still opens the buffer meny. It's just the `SPC` menus (before a "final" menu appears) that doesn't seem to work.

**Comment by simendsjo** [comment]
`M-m` doesn't open any menu either.

**Comment by StreakyCobra** [comment]
What's the value of `which-key-mode` (`SPC h d v which-key-mode`)? If it is `nil`, can you try turning it on with `M-x which-key-mode` and try to reproduce?

**Comment by simendsjo** [comment]
It was `nil`. Turning it on works! Not sure how I can turn it on always though.. Tried adding `(setq which-key-mode t)` in the config, but it's still `nil` after restarting.

**Comment by StreakyCobra** [comment]
It should be on by default, there is something that disable it in your config.

**Comment by syl20bnr** [comment]
To turn it on prefer to call the function instead of setting the variable:

```
(which-key-mode)
```

Alternatively you can use the spacemacs toggle which is bound under `SPC t`

```
(spacemacs/which-key-toggle-on)
```

**Comment by TheBB** [comment]
Nobody has been able to reproduce this, so I'm closing for now.

**Comment by LeaveNhA** [comment]
I have this issue now, I was using it without opening the menu but I needed it 2 min ago. It's not opening at all.
How can I debug the process?

**Comment by Irratzo** [comment]
For people still experiencing this issue, this issue has been discussed in issue #16276. In short, the issue has been fixed in pull request #16288. So, [updating](https://develop.spacemacs.org/doc/DOCUMENTATION.html#update-and-rollback) your Spacemacs/develop should fix it. Otherwise, see the most upvoted solution prior to the PR [here](https://github.com/syl20bnr/spacemacs/issues/16276#issuecomment-1939865121).
